airflow-core/src/airflow/dag_processing/dagbag.py

```diff
@@ -618,12 +618,12 @@ def collect_dags(
 
         files_to_parse = list_py_file_paths(dag_folder, safe_mode=safe_mode)
 
-        if include_examples:
-            from airflow import example_dags
+        # if include_examples:
```

Comment:
This change felt bold. Which is why I commented it out. If we were to let DagBag load examples, then we are duplicating the provider examples loading in both Dagbundle and in dagbag.

Will this affect backward compatibility?